app/models/offer.rb

```diff
@@ -1,4 +1,5 @@
 class Offer < ApplicationRecord
   belongs_to :user
```

Comment:
The belongs_to needs to be optional since scraped offers don't have a user. 

belongs_to :user, optional: true

https://www.bigbinary.com/blog/rails-5-makes-belong-to-association-required-by-default